src/bundle/Resources/public/scss/_pagination.scss

```diff
@@ -0,0 +1,41 @@
+.ez-pagination {
+	font-size: $pagination-font-size;
+}
+
+.page-item {
+	&:first-child {
+		.page-link {
+			border-radius: $pagination-border-radius;
+			background-color: $ez-color-secondary-hover;
+			color: $white;
+		}
+	}
+	&:last-child {
+		.page-link {
+			border-radius: $pagination-border-radius;
+			background-color: $ez-color-secondary-hover;
+			color: $white;
+		}
+	}
+
+	&.active .page-link {
+		border: 0 none;
+		font-weight: bold;
+		background-color: $pagination-active-bg;
+		color: $pagination-active-color;
+	}
+
+	&.disabled .page-link {
+		color: $pagination-disabled-color;
```

Comment:
Some of the rules still are copied from bootstrap. I think the reason if why you need it is because the bootstrap files are not regenerated. If you change the bootstrap variable you should regenerate the css with: `yarn sass-bootstrap`